### ISSUE TITLE
Suggestion: Make it easier to find this fork of data-diff?

### ISSUE THREAD

**Issue by Taytay** [erezsh/reladiff#49]
I came across articles about data-diff, got excited, and then saw that they'd archived it. I immediately started looking through their forks to see if anyone had picked up this project. (The most active one appears to be [this one](https://github.com/wealthsimple/data-diff)) It wasn't until I came across the [Reddit thread](https://www.reddit.com/r/dataengineering/comments/1dppemd/reladiff_highperformance_diffing_of_large/) that I stumbled across this repo! I am glad to see someone picking up the mantle (especially someone as invested as you). Good tabular data diff is sorely needed. I just wish I'd found it sooner. (Perhaps creating a real fork of it, or perhaps making it clearer from the outset that it was forked from data-diff? You know better than I do - just suggestions off the top of my head)

**Comment by erezsh** [comment]
Thanks for your concern, and I'm glad you find this project useful!

I did not fork data-diff directly, because reladiff diverges slightly from it. Also, Datafold asked me to keep the brand as separate as possible, and that's what I tried to do. However, I do mention data-diff in the readme, and elsewhere.

When I google "data-diff fork",  reladiff appears as the fifth result. Is it not the same way for you?

When I asked ChatGPT "what's the best fork of data-diff?", it replied "The most popular fork of the archived data-diff project is Reladiff"

I don't know if there's much more I can do, but I'm open to suggestions.

At the end of the day, I believe the best way to make the project seen is by making it popular. So star it if you haven't already, and maybe put a link to it somewhere, or post about it! Reladiff is still pretty new on the scene, and hopefully soon it will gain enough legitimacy to stand out even better.
